commit 48e39d6afd40de031c860bf920239fa850bc5d7c
Author: Jose Dapena Paz <jdapena@igalia.com>
Date:   Tue Jun 11 10:43:36 2024 +0000

    IWYU: missing include for usage of std::exchange in lock_impl.h
    
    Bug: 41455655
    Change-Id: If324eb8eece3dc710e6bcc032cc0587f45dd876b
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5616412
    Commit-Queue: Lei Zhang <thestig@chromium.org>
    Reviewed-by: Lei Zhang <thestig@chromium.org>
    Reviewed-by: Bartek Nowierski <bartekn@chromium.org>
    Cr-Commit-Position: refs/heads/main@{#1313273}

diff --git a/base/synchronization/lock_impl.h b/base/synchronization/lock_impl.h
index 6289b0a9a8f25..af4f268536aeb 100644
--- a/base/synchronization/lock_impl.h
+++ b/base/synchronization/lock_impl.h
@@ -5,6 +5,8 @@
 #ifndef BASE_SYNCHRONIZATION_LOCK_IMPL_H_
 #define BASE_SYNCHRONIZATION_LOCK_IMPL_H_
 
+#include <utility>
+
 #include "base/base_export.h"
 #include "base/check.h"
 #include "base/dcheck_is_on.h"
